fastseq_cli/generate.py

```diff
@@ -6,34 +6,25 @@
 by FastSeq.
 """
 
-from fairseq import options
 import os
-import argparse
+import sys
+
+from fairseq import options
 
-def parse_additional_args ():
+def parse_additional_args():
     parser = options.get_generation_parser()
-    parser.add_argument('--use-el-attn', action='store_true',
-            help='Use Efficient Lossless Attention optimization ? ')
     parser.add_argument(
-        '--postprocess-workers',
-        default=1,
-        type=int,
-        choices=range(1, 128, 1),
-        metavar='N',
-        help='number of worker for post process')
-    parser.add_argument(
-        '--decode-hypothesis',
-        action="store_true")
+        '--use-el-attn',
```

Comment:
Since we already used env variable to control on/off, seems no need to have an additional param for it. Just set this env variable outside should work well.

Reply:
Yes, env variable is used internally, so it should work for CLI as well.

Adding a parameter here aims to make the CLI easier to use for end users. What do you think?

Reply:
Ok.